### PR TITLE
Typos from web

### PR DESCRIPTION
**Pull request by Gelma**
These words are stolen from:
https://www.learnenglish.de/spelling/commonspellingmistakes.html

### CODE REVIEW

**Comment by larsoner** [comment]
That is not a failure, that passes. The failure is here:

https://travis-ci.org/codespell-project/codespell/jobs/402262986#L675

It says that `hight` is a duplicate entry

**Comment by Gelma** [comment]
@larsoner: thanks a lot! Really!
In the end I decided to write a stupid script to check dict before submit!
Now it should be fine!

**Comment by larsoner** [comment]
`pytest codespell_lib` would do this checking for you

**Comment by Gelma** [comment]
@larsoner: you are right, but my workflow is a little bit odd... So I preferred a custom solution. Good just for me, of course.

**Comment by larsoner** [comment]
@luzpaz do you have time to look first?

**Comment by luzpaz** [comment]
LGTM besides the issue I mentioned above

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3697,6 +3706,7 @@ diaplay->display
 diaplays->displays
 diappeares->disappears
 diaram->diagram, diorama,
+diarhea->diarrhoea
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Thanks a lot for the review. I'm going to upload new version.

**Pull request merged**